pages/comments.tsx

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import { inter } from "../styles/fonts";
+import { queryKeys } from "./queryKeys";
+import { useQuery } from "@tanstack/react-query";
+
+type Comment = {
+  id: number;
+  body: string;
+  name: string;
+  email: string;
+};
+const getComments = async () => {
+  const COMMENTS_URL = "https://jsonplaceholder.typicode.com/comments";
+  const res = await fetch(`${COMMENTS_URL}`);
+  return res.json();
+};
+function Comments() {
+  const { data, error, isLoading } = useQuery({
+    queryKey: [queryKeys.COMMENT],
+    queryFn: getComments,
+  });
```

Comment:
You were on the right track creating a separate hook for this. I did not ask you to remove that, I only asked you to incorporate the `queryKeys` thing into it

Reply:
ok